### PR TITLE
Fulfillment app translation

### PR DESCRIPTION
**Pull request by svohra1206**
Required translation are made.

### CODE REVIEW

File: FANewChanges.json [hunk 1]
```diff
@@ -0,0 +1,304 @@
+{
+  "All": "Todo",
+  "App": "Aplicación",
+  "A store represents a company or a unique catalog of products. If your OMS is connected to multiple eCommerce stores selling different collections of products, you may have multiple Product Stores set up in HotWax Commerce.": "Una tienda representa una empresa o un catálogo único de productos. Si tu OMS está conectado a múltiples tiendas de comercio electrónico que venden diferentes colecciones de productos, es posible que tengas varias Tiendas de Productos configuradas en HotWax Commerce.": "{facilityName} no tiene ningún pedido completado en este momento.",
+  "doesn't have any orders in progress right now.": "{facilityName} no tiene ningún pedido en curso en este momento.",
+  "doesn't have any outstanding orders right now.": "{facilityName} no tiene ningún pedido pendiente en este momento.",
+  "Add Box": "Agregar Caja",
+  "Add custom field": "Agregar campo personalizado",
+ "Additional documents": "Documentos adicionales",
```
Comment (ymaheshwari1) [inline FANewChanges.json:9]:
```suggestion
  "Additional documents": "Documentos adicionales",
```

File: FANewChanges.json [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+  "File uploaded successfully": "Archivo cargado con éxito",
+  "Full Address": "Dirección Completa",
+  "Fulfillment capacity": "Capacidad de cumplimiento",
+  "Generate packing slip": "Generar documento de embalaje",
+  "Generate shipping label": "Generar etiqueta de envío",
+  "Go to OMS": "Ir a OMS",
+  "Go to Launchpad": "Ir a Launchpad",
+  "is identified as unfulfillable. other containing this product will be unassigned from this store and sent to be rebrokered.": "{ productName } Se identifica como no cumplible  { space } { orders } otros { orderText } que contienen este producto se desasignarán de esta tienda y se enviarán para ser revendidos.",
+  "is identified as unfulfillable. This order item will be unassigned from this store and sent to be rebrokered.": "{ productName } se identifica como no cumplible. { espacio } Este elemento del pedido se desasignará de esta tienda y se enviará para ser revendido.",
+"is identified as. This order item will be unassigned from the store and sent to be rebrokered.": "{ productName } se identifica como { rejectReason }. Este elemento del pedido se desasignará de la tienda y se enviará para ser revendido.",
```
Comment (ymaheshwari1) [inline FANewChanges.json:102]:
```suggestion
  "is identified as. This order item will be unassigned from the store and sent to be rebrokered.": "{ productName } se identifica como { rejectReason }. Este elemento del pedido se desasignará de la tienda y se enviará para ser revendido.",
```

File: FANewChanges.json [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+  "Import Orders": "Importar Ordenes",
+  "Import shipped order details from an external system based on tracking codes. Orders that have tracking codes will automatically be shipped at the end of the day.": "Importa los detalles de los pedidos enviados desde un sistema externo basado en códigos de seguimiento. Los pedidos que tienen códigos de seguimiento se enviarán automáticamente al final del día.",
+  "Import shipped orders": "Importar pedidos enviados",
+  "In Progress": "En Curso",
+  "Key": "Clave",
+  "Language": "Lenguaje",
+  "Last brokered": "Última intermediación",
+  "Loading": "Cargando",
+  "Login": "Iniciar Sesión",
+   "Logging in": "Iniciando sesión",
```
Comment (ymaheshwari1) [inline FANewChanges.json:114]:
```suggestion
  "Logging in": "Iniciando sesión",
```

File: FANewChanges.json [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+  "Order packed successfully. Document generation in process": "Pedido empacado con éxito. Proceso de generación de documentos en curso",
+  "Order shipped successfully": "Pedido enviado exitosamente",
+  "Order Shipment ID": "ID de Envío del Pedido",
+  "Order unpacked successfully": "Pedido desempacado exitosamente",
+  "Order updated successfully": "Pedido actualizado exitosamente",
+  "orders": "pedidos",
+  "Out of stock": "Agotado",
+  "package": "paquete",
+  "packages": "paquetes",
+ "Pack": "Empacar",
```
Comment (ymaheshwari1) [inline FANewChanges.json:163]:
```suggestion
  "Pack": "Empacar",
```

File: FANewChanges.json [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+  "Ready to pack": "Listo para empacar",
+  "Reject": "Rechazar",
+  "Reject all": "Rechazar todo",
+  "Reject all in progress orders": "Rechazar todos los pedidos en curso",
+  "Reject all open orders": "Rechazar todos los pedidos abiertos",
+  "Reject in progress orders.": "Rechazar { ordersCount } pedidos en curso.",
+  "Reject open orders.": "Rechazar { ordersCount } pedidos abiertos.",
+  "Reject order": "Rechazar pedido",
+  "Rejecting has been started. All in progress orders will be rejected shortly.": "Se ha iniciado el proceso de rechazo. Todos los pedidos en curso se rechazarán en breve.",
+ "Rejecting has been started. All outstanding orders will be rejected shortly.": "Se ha iniciado el proceso de rechazo. Todos los pedidos pendientes se rechazarán en breve.",
```
Comment (ymaheshwari1) [inline FANewChanges.json:201]:
```suggestion
  "Rejecting has been started. All outstanding orders will be rejected shortly.": "Se ha iniciado el proceso de rechazo. Todos los pedidos pendientes se rechazarán en breve.",
```

File: FANewChanges.json [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+  "Select your preferred language.": "Seleccione .",
+  "Selected Fields: ": "Filas seleccionadas: ",
+  "Sell inventory online": "Vender inventario en línea",
+  "Sell online": "Vender en línea",
+  "Setting fulfillment capacity to 0 disables new order from being allocated to this facility. Leave this empty if this facility's fulfillment capacity is unrestricted.": "Establecer la capacidad de cumplimiento en 0 inhabilita la asignación de nuevos pedidos a esta instalación. Déjelo vacío si la capacidad de cumplimiento de esta instalación no tiene restricciones.",
+  "Settings": "Configuraciones",
+  "Set Limit": "Establecer Límite",
+  "Ship": "Enviar",
+  "Shipments ready to ship": "Envíos listos para enviar",
+"Ship Now": "Enviar Ahora",
```
Comment (ymaheshwari1) [inline FANewChanges.json:238]:
```suggestion
  "Ship Now": "Enviar Ahora",
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: FANewChanges.json [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+  "Change": "Cancelar",
+  "Changes to the CSV mapping has been saved.": "Los cambios en el mapeo CSV se han guardado.",
+  "Choose language": "Elegir idioma",
+  "City": "Ciudad",
+  "Completed": "Completado",
+  "Configuration Missing": "Configuración Faltante",
+  "Confirm": "Confirmar",
+  "Configure the order fulfillment capacity of your facility.": "Configura la capacidad de cumplimiento de pedidos de tu instalación.",
+  "Control whether the store's inventory should be made available for online sales or not.": "Controla si el inventario de la tienda debe estar disponible para ventas en línea o no.",
+  "Copied": "Copiado {texto}",
```
Comment (ymaheshwari1) [inline FANewChanges.json:34]: Revert this variable name change.

File: FANewChanges.json [hunk 8]
```diff
@@ -0,0 +1,304 @@
...
+  "File downloaded successfully": "Archivo descargado con éxito",
+  "File uploaded successfully": "Archivo cargado con éxito",
+  "Full Address": "Dirección Completa",
+  "Fulfillment capacity": "Capacidad de cumplimiento",
+  "Generate packing slip": "Generar documento de embalaje",
+  "Generate shipping label": "Generar etiqueta de envío",
+  "Go to OMS": "Ir a OMS",
+  "Go to Launchpad": "Ir a Launchpad",
+  "is identified as unfulfillable. other containing this product will be unassigned from this store and sent to be rebrokered.": "{ productName } Se identifica como no cumplible  { space } { orders } otros { orderText } que contienen este producto se desasignarán de esta tienda y se enviarán para ser revendidos.",
+  "is identified as unfulfillable. This order item will be unassigned from this store and sent to be rebrokered.": "{ productName } se identifica como no cumplible. { espacio } Este elemento del pedido se desasignará de esta tienda y se enviará para ser revendido.",
```
Comment (ymaheshwari1) [inline FANewChanges.json:101]:
```suggestion
  "is identified as unfulfillable. This order item will be unassigned from this store and sent to be rebrokered.": "{ productName } se identifica como no cumplible. { space } Este elemento del pedido se desasignará de esta tienda y se enviará para ser revendido.",
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: FANewChanges.json (hunk 7 above)
Comment (Dhiraj1405) [reply FANewChanges.json:34]: which exact variable are we talking about here, I mostly see each of them has translations.

Comment (ymaheshwari1) [reply FANewChanges.json:34]: `texto` should be `text`, anything that's inside `{}` are variables and not need to be changed.

File: FANewChanges.json [hunk 9]
```diff
@@ -0,0 +1,304 @@
+{
+  "All": "Todo",
+  "App": "Aplicación",
+  "A store represents a company or a unique catalog of products. If your OMS is connected to multiple eCommerce stores selling different collections of products, you may have multiple Product Stores set up in HotWax Commerce.": "Una tienda representa una empresa o un catálogo único de productos. Si tu OMS está conectado a múltiples tiendas de comercio electrónico que venden diferentes colecciones de productos, es posible que tengas varias Tiendas de Productos configuradas en HotWax Commerce.": "{facilityName} no tiene ningún pedido completado en este momento.",
```
[the remark this answers is not in the document]
Comment (ymaheshwari1) [reply FANewChanges.json:4]: Yes, we need this.

```suggestion
  "A store represents a company or a unique catalog of products. If your OMS is connected to multiple eCommerce stores selling different collections of products, you may have multiple Product Stores set up in HotWax Commerce.": "Una tienda representa una empresa o un catálogo único de productos. Si tu OMS está conectado a múltiples tiendas de comercio electrónico que venden diferentes colecciones de productos, es posible que tengas varias Tiendas de Productos configuradas en HotWax Commerce.",
  "doesn't have any completed orders right now.": "{facilityName} no tiene ningún pedido completado en este momento.",
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: FANewChanges.json [hunk 10]
```diff
@@ -0,0 +1,305 @@
+{
```
Comment (ymaheshwari1) [inline FANewChanges.json:1]: Change file name to `es.json`

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Comment by ymaheshwari1** [comment]
Closing as handled in https://github.com/hotwax/fulfillment-pwa/pull/333

**Pull request closed without merging**